CHANGELOG.md

```diff
@@ -1,4 +1,35 @@
-## 4.6.0 (Unreleased)
+## 4.6.0 (January 10, 2021)
+
+BREAKING CHANGES:
+* pubsub: changed `google_pubsub_schema` so that modifiying fields will recreate the resource rather than causing Terraform to report it would attempt an invalid update ([#10768](https://github.com/hashicorp/terraform-provider-google/pull/10768))
```

Comment:
Note: this shouldn't actually break anyone. See https://github.com/GoogleCloudPlatform/magic-modules/pull/5553#pullrequestreview-834440605